api/src/main/java/org/bahmni/module/hwcinventory/contract/CreatePatientRequest.java

```diff
@@ -0,0 +1,135 @@
+package org.bahmni.module.hwcinventory.contract;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CreatePatientRequest {
+    private String abhaAddress;
+    private String abhaNumber;
+    private int age;
+    private String birthdate;
+    private String displayName;
+    private String firstName;
+    private String middleName;
+    private String lastName;
+    private int genderCode;
+    private String genderDisplay;
+    private boolean isBlock;
+    private List<EsanjeevaniPatientAddress> lstPatientAddress;
```

Comment:
What does "lst" mean?

Reply:
@gsluthra this is list of patient Address, we cannot change it as request body will only accept this.